<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>REGISTRO DEL ALMACEN</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="menuizquierdo">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="menubotones">
               <property name="styleSheet">
                <string notr="true">QFrame{
background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:0.06, y2:0.131227, stop:0.373134 rgba(249, 148, 207, 255), stop:1 rgba(255, 255, 255, 255))
}

QPushButton{
border:2px solid rgb(255, 85, 127);
border-radius:20px;

}

QPushButton:hover{
background-color:rgb(255, 255, 255);
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <widget class="QPushButton" name="btnRegistro">
                <property name="geometry">
                 <rect>
                  <x>30</x>
                  <y>110</y>
                  <width>121</width>
                  <height>51</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Verdana</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>registro</string>
                </property>
               </widget>
               <widget class="QPushButton" name="btnPag2">
                <property name="geometry">
                 <rect>
                  <x>30</x>
                  <y>190</y>
                  <width>121</width>
                  <height>51</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Verdana</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>pag2</string>
                </property>
               </widget>
               <widget class="QPushButton" name="btnPag3">
                <property name="geometry">
                 <rect>
                  <x>30</x>
                  <y>280</y>
                  <width>131</width>
                  <height>51</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Verdana</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>pag3</string>
                </property>
               </widget>
               <widget class="QPushButton" name="btnPag4">
                <property name="geometry">
                 <rect>
                  <x>30</x>
                  <y>370</y>
                  <width>121</width>
                  <height>41</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Verdana</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>pag4</string>
                </property>
               </widget>
               <widget class="QPushButton" name="btnPag5">
                <property name="geometry">
                 <rect>
                  <x>30</x>
                  <y>440</y>
                  <width>121</width>
                  <height>41</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Verdana</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>pag5</string>
                </property>
               </widget>
               <widget class="QPushButton" name="btnSalir">
                <property name="geometry">
                 <rect>
                  <x>30</x>
                  <y>790</y>
                  <width>131</width>
                  <height>51</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Verdana</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>salir</string>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="menuderecho">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QStackedWidget" name="paginas">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="page">
                <widget class="QPushButton" name="pushButton_7">
                 <property name="geometry">
                  <rect>
                   <x>290</x>
                   <y>710</y>
                   <width>151</width>
                   <height>51</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>guardar</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_8">
                 <property name="geometry">
                  <rect>
                   <x>500</x>
                   <y>710</y>
                   <width>151</width>
                   <height>51</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>modificar</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_10">
                 <property name="geometry">
                  <rect>
                   <x>720</x>
                   <y>710</y>
                   <width>151</width>
                   <height>51</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>guardar</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_21">
                 <property name="geometry">
                  <rect>
                   <x>120</x>
                   <y>20</y>
                   <width>511</width>
                   <height>121</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Tw Cen MT Condensed Extra Bold</family>
                   <pointsize>48</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>BIENVENIDO </string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
                <widget class="QLabel" name="label_22">
                 <property name="geometry">
                  <rect>
                   <x>60</x>
                   <y>190</y>
                   <width>621</width>
                   <height>91</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>24</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>VENTANA DE ALMACEN</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
                <widget class="QLabel" name="label_23">
                 <property name="geometry">
                  <rect>
                   <x>240</x>
                   <y>360</y>
                   <width>241</width>
                   <height>261</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>logo1.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </widget>
               <widget class="QWidget" name="page_2">
                <widget class="QLineEdit" name="lineEdit_7">
                 <property name="geometry">
                  <rect>
                   <x>410</x>
                   <y>250</y>
                   <width>161</width>
                   <height>41</height>
                  </rect>
                 </property>
                </widget>
                <widget class="QLabel" name="label_8">
                 <property name="geometry">
                  <rect>
                   <x>260</x>
                   <y>430</y>
                   <width>111</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Apellido</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_9">
                 <property name="geometry">
                  <rect>
                   <x>260</x>
                   <y>350</y>
                   <width>111</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>nombre</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_10">
                 <property name="geometry">
                  <rect>
                   <x>260</x>
                   <y>250</y>
                   <width>111</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>cod_cliente</string>
                 </property>
                </widget>
                <widget class="QLineEdit" name="lineEdit_8">
                 <property name="geometry">
                  <rect>
                   <x>410</x>
                   <y>430</y>
                   <width>161</width>
                   <height>41</height>
                  </rect>
                 </property>
                </widget>
                <widget class="QLineEdit" name="lineEdit_9">
                 <property name="geometry">
                  <rect>
                   <x>410</x>
                   <y>340</y>
                   <width>161</width>
                   <height>41</height>
                  </rect>
                 </property>
                </widget>
               </widget>
               <widget class="QWidget" name="page_3">
                <widget class="QLineEdit" name="lineEdit_10">
                 <property name="geometry">
                  <rect>
                   <x>700</x>
                   <y>50</y>
                   <width>161</width>
                   <height>41</height>
                  </rect>
                 </property>
                </widget>
                <widget class="QLabel" name="label_11">
                 <property name="geometry">
                  <rect>
                   <x>550</x>
                   <y>230</y>
                   <width>111</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Apellido</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_12">
                 <property name="geometry">
                  <rect>
                   <x>550</x>
                   <y>150</y>
                   <width>111</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>nombre</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_13">
                 <property name="geometry">
                  <rect>
                   <x>550</x>
                   <y>50</y>
                   <width>111</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>cod_cliente</string>
                 </property>
                </widget>
                <widget class="QLineEdit" name="lineEdit_11">
                 <property name="geometry">
                  <rect>
                   <x>700</x>
                   <y>230</y>
                   <width>161</width>
                   <height>41</height>
                  </rect>
                 </property>
                </widget>
                <widget class="QLineEdit" name="lineEdit_12">
                 <property name="geometry">
                  <rect>
                   <x>700</x>
                   <y>140</y>
                   <width>161</width>
                   <height>41</height>
                  </rect>
                 </property>
                </widget>
               </widget>
               <widget class="QWidget" name="page_4">
                <widget class="QLineEdit" name="lineEdit_13">
                 <property name="geometry">
                  <rect>
                   <x>240</x>
                   <y>550</y>
                   <width>161</width>
                   <height>41</height>
                  </rect>
                 </property>
                </widget>
                <widget class="QLabel" name="label_14">
                 <property name="geometry">
                  <rect>
                   <x>90</x>
                   <y>730</y>
                   <width>111</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Apellido</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_15">
                 <property name="geometry">
                  <rect>
                   <x>90</x>
                   <y>650</y>
                   <width>111</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>nombre</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_16">
                 <property name="geometry">
                  <rect>
                   <x>90</x>
                   <y>550</y>
                   <width>111</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>cod_cliente</string>
                 </property>
                </widget>
                <widget class="QLineEdit" name="lineEdit_14">
                 <property name="geometry">
                  <rect>
                   <x>240</x>
                   <y>730</y>
                   <width>161</width>
                   <height>41</height>
                  </rect>
                 </property>
                </widget>
                <widget class="QLineEdit" name="lineEdit_15">
                 <property name="geometry">
                  <rect>
                   <x>240</x>
                   <y>640</y>
                   <width>161</width>
                   <height>41</height>
                  </rect>
                 </property>
                </widget>
               </widget>
               <widget class="QWidget" name="page_5">
                <widget class="QLineEdit" name="lineEdit_16">
                 <property name="geometry">
                  <rect>
                   <x>260</x>
                   <y>70</y>
                   <width>161</width>
                   <height>41</height>
                  </rect>
                 </property>
                </widget>
                <widget class="QLabel" name="label_17">
                 <property name="geometry">
                  <rect>
                   <x>110</x>
                   <y>250</y>
                   <width>111</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Apellido</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_18">
                 <property name="geometry">
                  <rect>
                   <x>110</x>
                   <y>170</y>
                   <width>111</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>nombre</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_19">
                 <property name="geometry">
                  <rect>
                   <x>110</x>
                   <y>70</y>
                   <width>111</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <family>Verdana</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>cod_cliente</string>
                 </property>
                </widget>
                <widget class="QLineEdit" name="lineEdit_17">
                 <property name="geometry">
                  <rect>
                   <x>260</x>
                   <y>250</y>
                   <width>161</width>
                   <height>41</height>
                  </rect>
                 </property>
                </widget>
                <widget class="QLineEdit" name="lineEdit_18">
                 <property name="geometry">
                  <rect>
                   <x>260</x>
                   <y>160</y>
                   <width>161</width>
                   <height>41</height>
                  </rect>
                 </property>
                </widget>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
